### PR TITLE
Signup form

### PR DESCRIPTION
**Pull request by jstloyal**
The HTML form is a collaborative project that consists of building an HTML document that matches the appearance of mint.com’s signup page. 
mysignup-form

### CODE REVIEW

**Comment by majovanilla** [comment]
# Code Review Request in progress 👀 by Maria Reyes

**Comment by majovanilla** [comment]
Hey @jstloyal and @harshdeepkanhai , 👋

## Good job on this project! 👍

There are still some changes 👇 you need to implement to approve this request 🔍

### Required Changes: ✍️

- You need to improve your READMe file. It doesn't contain enough information about the project and some sections have mock subtitles. Try using the main requirements of the project as an introduction.

- Add the logos on top of your form, like in the original page. You don't need to add the effect.
Screenshot from 2020-02-21 02-52-18

- There are some styling differences between the original page. Remember it has to match it. This is a Microverse requirement. See screenshot here: 
Screenshot from 2020-02-21 02-57-51

      - Use different background color for the upper part of the form.
      - Use a similar size for the inputs.
      - Use a similar style for the inputs (border).
      - Use a similar font size.
      - Change the color of the links to match the original page.

### Optional Changes:

If you want to make your project even better 🔥💥, I have a few suggestions you can implement.  

These are completely [Optional]: 🦔

- Use semantic HTML at least for the main sections (include the header tag instead of div for the top section)
Screenshot from 2020-02-21 02-55-55


## Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack (@majo)
I would be happy to help! 👩‍💻 


Happy coding 🤓

--Maria Reyes



You can find me on Social Media below 👇 . Remember to say hi!

[GitHub](https://github.com/majovanilla) [Twitter](https://twitter.com/MajoVanilla) [LinkedIn](https://www.linkedin.com/in/majoreyesparroquin/)

**Comment by majovanilla** [comment]
As an additional tip, you can use this page: http://raw.githack.com/ to add a live version of your project in the ReadMe file.

**Comment by pelzolga123** [comment]
## Hi Team!  
### Your project looks :+1: amazing!

Just a few changes left before this project can be approved:
* [ ] check HTML markup with this [validator](https://validator.w3.org/#validate_by_upload). Now it shows errors in your project.
* [ ] Your project is missing disclaimer text from original website, you can find it under phone input:
image

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Comment by jstloyal** [comment]
Thank you for the wonderful review!

**Comment by majovanilla** [comment]
# Code  under review 👀 by Maria Reyes

**Comment by majovanilla** [comment]
Hey Team, 👋

## Good job on this project! 👍

You are almos there!! Remember to stick with a general checking order so you don't miss stuff like this:
There are still some changes 👇 you need to implement to approve this request 🔍

### Required Changes: ✍️

- HTML validator still has errors and warnings.
Screenshot from 2020-02-21 18-25-18


You can do this!! It's almost done!! :muscle:

**Comment by jstloyal** [comment]
I have added some meta element as advised by majovanilla and I hope it's all done now!

File: index.html [hunk 1]
```diff
@@ -1,9 +1,81 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <title>Contributing to Open Source Part 1</title>
+    <title>Mint Sign up page</title>
     <link href="./css/main.css" rel="stylesheet" type="text/css" />
     <meta charset="UTF-8" />
+    <meta value="https://github.com/harshdeepkanhai/mint.com-signupage-clone/" />
```
Comment (uryelah) [inline index.html]: You don't really need this tag in the head. It's the only thing causing HTML validation errors right now, please delete it and you'll be done. Remember to ALWAYS check your HTML code with the [validator ](https://validator.w3.org/nu/) before submitting it for a code review.

**Comment by lekegitrepo** [comment]
This PR is under Review by: **[TSE] Temitayo Adeleke**

File: index.html (hunk 1 above)
Comment (jstloyal) [reply index.html]: Thank you, I have removed the tag!

**Comment by lekegitrepo** [comment]
### Hi Team,
Well done, this **_project is approved_**.

Reviewed by: Temitayo Adeleke

**Comment by jstloyal** [comment]
Thank you!

**Pull request merged**